docs/wearables/sdks/flutter.mdx

```diff
@@ -136,6 +136,27 @@ Finally, observe the status stream, so you can react to changes:
 Stream<SyncStatus> status = client.healthServices.status;
 ```
 
+### 3. Background Delivery
+
+Setting up background syncing in Flutter follows the [same approach as native iOS](/wearables/sdks/ios/). You need to configure `VitalHealthKit` in the AppDelegate, otherwise background syncing won't work.
+
+You can do so by calling:
+
+```swift
+import VitalHealthKit
+
+VitalHealthKitClient.configure(
+    .init(
+        mode: ..., // automatic by default
+        backgroundDeliveryEnabled: ..., // false by default
+        logsEnabled: ..., // false by default
+        numberOfDaysToBackFill: ..., // 90 by default
+    )
+)
+```
+
+You can also check our [Flutter example](https://github.com/tryVital/vital-flutter/blob/main/example/ios/Runner/AppDelegate.swift#L11C1-L12C1). In our example, we use the `VitalHealthKitClient.automaticConfiguration()` variant. This version assumes that `VitalHealthKitClient.configure` was called at some point in the past (e.g. when the user was logged in). For example,`automaticConfiguration` is helpful when you are doing A/B testing and you only want to enroll certain users with Apple HealthKit.
```

Comment:
Should we not recommend `automaticConfiguration` by default?

Reply:
We would also need to update the iOS docs, since we don't mention this anywhere.

Reply:
what I mean is that our SDK currently does support a Flutter/RN customer to asynchronously configure VitalClient & VitalHealthKit via the Dart/TypeScript, and only having to do `automaticConfiguration()` in AppDelegate didFinishLaunching.

This might be a more appealing recommendation?

Reply:
We can. but the only reason why we mention automatic configuration, is because that's what we have in the Flutter example. I will create a new ticket to address that.